modules/ROOT/pages/migrate-from-an-earlier-version-of-bonita.adoc

```diff
@@ -301,7 +294,6 @@ Install the new license:
 
 . Start the application server. Before you start Bonita Portal, clear your browser cache. If you do not clear the cache, you might see old, cached versions of Portal pages instead of the new version.
  Log in to the Portal and verify that the migration has completed.
```

Comment:
Log in to the Portal and verify that the migration has completed. => Log in to Bonita Administrator Application and verify that the update status.

Reply:
The context of the phrase is before 2021.2, Bonita Portal still exists